docs/docs/recipes/manage-users/on-admin-console-page.md

```diff
@@ -7,7 +7,18 @@ sidebar_position: 1
 
 ## Add user
```

Comment:
better make them as an ordered or unordered bullet list?

Reply:
Each section seems to be a little longer.
Maybe they're not suitable to be contained in an ordered or unordered bullet list.

We can reorg them after finishing their content later.